pom.xml

```diff
@@ -75,6 +75,7 @@
     <module>flink-connector-akka</module>
     <module>flink-connector-flume</module>
     <module>flink-connector-influxdb</module>
+    <module>flink-connector-influxdb2</module>
```

Comment:
why create another module?
the first one could not be updated?

Reply:
Unfriendly not. Since there are breaking changes in InfluxDB 2.x. Even the java client is written from scratch and it's not compatible with version 1.7 anymore. More information [here](https://github.com/influxdata/influxdb-client-java#note-use-this-client-library-with-influxdb-2x-and-influxdb-18-see-details-for-connecting-to-influxdb-17-or-earlier-instances-use-the-influxdb-java-client-library). We created a separate module to avoid a situation where other users still rely on InfluxDB 1.x connectors.

Reply:
I guess yes, because we implemented influxDB 2.X which is still quite new. I guess a lot of people still use the old api.